src/si/momentum.rs

```diff
@@ -0,0 +1,201 @@
+//! Momentum (base unit kilogram meter per second, kg<sup>1</sup> · m<sup>1</sup> · s<sup>-1</sup>).
+
+quantity! {
+    /// Momentum (base unit kilogram meter per second, kg<sup>1</sup> · m<sup>1</sup> · s<sup>-1</sup>).
```

Comment:
Wrap at the first space before the 100 character mark.

Reply:
done